src/docs/patterns/visualizations.mdx

```diff
@@ -1,62 +1,93 @@
-
-> **NOTE:** the following examples merely serve to show how to bring the charting libraries into your application and adjust some of the common chart parameters. You will want to adjust these values to meet the design requirements of your application.
-
-### Highcharts
+> The following examples merely serve to show how to bring the charting libraries into your application and adjust some of the common chart parameters. You will want to adjust these values to meet the design requirements of your application.
 
 <DemoCard
     repository={'highcharts'}
     angular={`https://stackblitz.com/github/pxblue/highcharts/tree/master/angular-demo`}
     react={`https://codesandbox.io/s/github/pxblue/highcharts/tree/master/react-demo`}
 />
-
-PX Blue utilities for Highcharts graphs (line, column, pie, donut) are available as an NPM package ([@pxblue/highcharts](https://www.npmjs.com/package/@pxblue/highcharts)).
-This package includes helper functions to generate chart configurations using recommended PX Blue styles.
-These utilities also enable you to quickly create placeholder charts using simulated data. For more information about using Highcharts, check out their documentation.
-
-> Highcharts has different [licensing](https://shop.highsoft.com/developer-license) requirements depending on the nature of your application - you will need to contact Highsoft directly to ensure that your product is properly licensed. You can also see the [Highcharts Pricing](https://shop.highsoft.com/)) page for pricing information.
-
-### ChartJS
-
 <DemoCard
     repository={'chartjs'}
     angular={`https://stackblitz.com/github/pxblue/chartjs/tree/master/angular-demo`}
     react={`https://codesandbox.io/s/github/pxblue/chartjs/tree/master/react-demo`}
 />
 
-ChartJS is an open source alternative to HighCharts offering a scaled-down feature set that supports basic charting needs.
+## Common Pitfalls
+
+<ImageGrid images={[ClockwisePie]} regularWidth caption={'A pie chart should always be clockwise.'} />
 
-PX Blue utilities for ChartJS graphs (line, column, pie, donut) are available as an NPM package ([@pxblue/chartjs](https://www.npmjs.com/package/@pxblue/chartjs)). For more information, check out the [ChartJS Documentation](https://www.chartjs.org/).
+To avoid interfering with other UI elements, the charts should be built using primarily the [branding colors](/style/color#branding-colors).
 
-## Mapping (Mapbox)
+<React.Fragment>
+    <ImageGrid images={[DontUsePrimaryColor]} regularWidth caption={'Never use primary color palette in the charts.'} />
+    <Divider />
+</React.Fragment>
 
+# Maps
+
+<ImageGrid
+    regularWidth
+    images={[MapsImg]}
+    caption={[
+        'An example of a map, built with PX Blue dark theme.',
+        'The embedded map is a screenshot from OpenStreetMap, licensed under CC-BY-SA 2.0.',
```

Comment:
Do we need this caption at all?

Or maybe just "A simple geographic map using the dark theme."

Reply:
That credit line is necessary under the cc-by-sa license... 
The first line is updated to what you suggested.